tests/rt_wcoss_cray.conf

```diff
@@ -1,68 +0,0 @@
-#######################################################################################################################################################################################
```

Comment:
This file is deleted but in rt.sh there's still an if test that checks if we are on wcoss_cray and resets TESTS_FILE to rt_wcoss_cray.conf.

Reply:
Thanks for catching this! I remember `tests/rt_wcoss_cray.conf` was added because IPD REPRO, CCPP REPRO and CCPP PROD weren't run there. This code *should* run on wcoss_cray and produce b4b identical results, in which case the extra file `tests/rt_wcoss_cray.conf` won't be needed. Two options: (a) change `rt.sh` to use the standard `rt.conf` on wcoss_cray and test if this now works as expected, or (b) revert back deleting `tests/rt_wcoss_cray.conf`. What do you prefer? On the long run we need to get to option (a), though.

Reply:
I tentatively made the change following option (a).

Reply:
Agree. Option (a) is preferred.

Reply:
Ok, let's try. I am getting close to having the move from ./ccpp to ./FV3/ccpp done, including most of the out-of-source CCPP build.